stdlib/src/builtin/string.mojo

```diff
@@ -211,6 +211,198 @@ fn ascii(value: String) -> String:
 # ===----------------------------------------------------------------------=== #
 
 
+@always_inline
+fn _stol(str_ref: StringRef, base: Int = 10) raises -> (Int, StringRef):
```

Comment:
Could you change this to use `StringSlice` ? we are migrating away from `StringRef`